Not working yet; for review

DaveSeidel opened the pull request:
I made an override for onPortChange(), and it's being called, but the ports are not being reset as expected. Clearly I'm missing something important (not surprising, since I've never worked with this API before).

DaveSeidel committed:
Not working yet; for review

docb commented:
i think we do not need to check this port event, as the method setParameters is anyway called every k-cycle,
i would fix it there.

DaveSeidel commented:
Ok I will try that.

On Mon, Jan 9, 2023, 1:37 PM docb ***@***.***> wrote:

> i think we do not need to check this port event, as the method
> setParameters is anyway called every k-cycle,
> i would fix it there.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/docb/dbRackCsound/pull/10#issuecomment-1376102555>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABTVZLXGGHSPILWNVSKLRTWRRLG3ANCNFSM6AAAAAATVXWHT4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

docb commented:
you dont't need to i have fixed that btw when i introduced the connecction status channels,

see here:
31b7cd928f72353a172394d07fc2e3b91177bec4

DaveSeidel commented:
Great! Will look soon.

On Mon, Jan 9, 2023, 2:04 PM docb ***@***.***> wrote:

> you dont't need to i have fixed that btw when i introduced the connecction
> status channels,
>
> see here:
> 31b7cd9
> <https://github.com/docb/dbRackCsound/commit/31b7cd928f72353a172394d07fc2e3b91177bec4>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/docb/dbRackCsound/pull/10#issuecomment-1376139899>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABTVZIOEBUH2LL4TQNYRBDWRRONPANCNFSM6AAAAAATVXWHT4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

DaveSeidel commented:
This is working great now with your last push. Thanks very much!

On Mon, Jan 9, 2023 at 2:06 PM Dave Seidel ***@***.***> wrote:

> Great! Will look soon.
>
> On Mon, Jan 9, 2023, 2:04 PM docb ***@***.***> wrote:
>
>> you dont't need to i have fixed that btw when i introduced the
>> connecction status channels,
>>
>> see here:
>> 31b7cd9
>> <https://github.com/docb/dbRackCsound/commit/31b7cd928f72353a172394d07fc2e3b91177bec4>
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/docb/dbRackCsound/pull/10#issuecomment-1376139899>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AABTVZIOEBUH2LL4TQNYRBDWRRONPANCNFSM6AAAAAATVXWHT4>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
